hack/ci-e2e.sh

```diff
@@ -105,6 +87,19 @@ else
   exit 1
 fi
 
+export E2E_BMCS_CONF_FILE="${REPO_ROOT}/test/e2e/config/bmcs-${BMO_E2E_EMULATOR}.yaml"
+"${REPO_ROOT}/hack/create_bmcs.sh" "${E2E_BMCS_CONF_FILE}" baremetal-e2e
+
+# Set the number of ginkgo processes to the smaller number in
+# the number of host CPUs and the number of BMCs
+n_cpus=$(nproc)
```

Comment:
Not sure this is a good idea. Each VM we create has 2 vcpu so we would quickly use up all the CPU I think :smile:

Reply:
Yeah, perhaps just use all the VMs then? Or do you think we should limit it to 2 only?

Reply:
Using all the VMs sounds good. I would also be happy with not doing anything at all, i.e. just pass through. Then we can just set the variable before calling the script